crates/sb_fs/file_system.rs

```diff
@@ -33,22 +35,25 @@ impl DenoCompileFileSystem {
         }
     }
 
-    fn copy_to_real_path(&self, oldpath: &Path, newpath: &Path) -> FsResult<()> {
```

Comment:
why removing this function? We could just create an async one?

Reply:
I recall that some of the functions it calls in its function body had their signatures changed to async, so I changed those functions to async due to contagious.

Reply:
I guess I could probably make that synchronously again by spawning async functions in the blocking threads